test/distributed/org/apache/cassandra/fuzz/topology/AccordTopologyMixupTest.java

```diff
@@ -0,0 +1,397 @@
+import org.apache.cassandra.config.Config;
+import org.apache.cassandra.cql3.ast.Mutation;
+import org.apache.cassandra.cql3.ast.Statement;
+import org.apache.cassandra.distributed.Cluster;
+import org.apache.cassandra.distributed.api.ConsistencyLevel;
+import org.apache.cassandra.distributed.api.IInstanceConfig;
+import org.apache.cassandra.distributed.api.IInvokableInstance;
+import org.apache.cassandra.distributed.api.SimpleQueryResult;
+import org.apache.cassandra.distributed.shared.ClusterUtils;
+import org.apache.cassandra.distributed.test.accord.AccordTestBase;
+import org.apache.cassandra.schema.TableMetadata;
+import org.apache.cassandra.service.accord.api.AccordAgent;
+import org.apache.cassandra.service.accord.api.PartitionKey;
+import org.apache.cassandra.service.consensus.TransactionalMode;
+import org.apache.cassandra.utils.ASTGenerators;
+import org.apache.cassandra.utils.CassandraGenerators;
+import org.apache.cassandra.utils.Isolated;
+import org.apache.cassandra.utils.JsonUtils;
+import org.apache.cassandra.utils.Shared;
+
+import static org.apache.cassandra.utils.AccordGenerators.fromQT;
+
+public class AccordTopologyMixupTest extends TopologyMixupTestBase<AccordTopologyMixupTest.Spec>
+{
+    private static final Logger logger = LoggerFactory.getLogger(AccordTopologyMixupTest.class);
+
+    static
+    {
+        CassandraRelevantProperties.ACCORD_AGENT_CLASS.setString(InterceptAgent.class.getName());
+        // enable most expensive debugging checks
+        CassandraRelevantProperties.ACCORD_KEY_PARANOIA_CPU.setString(Invariants.Paranoia.QUADRATIC.name());
+        CassandraRelevantProperties.ACCORD_KEY_PARANOIA_MEMORY.setString(Invariants.Paranoia.QUADRATIC.name());
+        CassandraRelevantProperties.ACCORD_KEY_PARANOIA_COSTFACTOR.setString(Invariants.ParanoiaCostFactor.HIGH.name());
+    }
+
+    private static final List<TransactionalMode> TRANSACTIONAL_MODES = Stream.of(TransactionalMode.values()).filter(t -> t.accordIsEnabled).collect(Collectors.toList());
+
+    @Override
+    protected Gen<State<Spec>> stateGen()
+    {
+        return AccordState::new;
+    }
+
+    @Override
+    protected void preCheck(Property.StatefulBuilder builder)
+    {
+        // if a failing seed is detected, populate here
+        // Example: builder.withSeed(42L);
+    }
+
+    private static Spec createSchemaSpec(RandomSource rs, Cluster cluster)
+    {
+        TransactionalMode mode = rs.pick(TRANSACTIONAL_MODES);
+        boolean enableMigration = allowsMigration(mode) && rs.nextBoolean();
+        TableMetadata metadata = fromQT(new CassandraGenerators.TableMetadataBuilder()
+                                        .withKeyspaceName(KEYSPACE)
+                                        .withTableKinds(TableMetadata.Kind.REGULAR)
+                                        .withKnownMemtables()
+                                        //TODO (coverage): include "fast_path = 'keyspace'" override
```

Comment:
this is a method now in `https://issues.apache.org/jira/browse/CASSANDRA-18675` and found bugs with our fast path serializers...